packages/swingset-liveslots/src/liveslots.js

```diff
@@ -1338,7 +1335,9 @@ function build(
       makeScalarBigSetStore: collectionManager.makeScalarBigSetStore,
       makeScalarBigWeakSetStore: collectionManager.makeScalarBigWeakSetStore,
     },
-  });
+    [PassStyleOfEndowmentSymbol]: passStyleOf,
+  };
+  harden(vatGlobals);
```

Comment:
No objection to doing the `harden` here. But curious about your motivation for moving it? There's no mutation between the initialization and the harden, so the more traditional harden around the object literal should still be equivalent.

Reply:
I don't remember why I moved it.. I'll move it back.